### PR TITLE
Deploy author from Monorepo

### PR DESCRIPTION
**Pull request by jonnyshaw89**
- Staging deployed on master merge
 - PreProd deployed on GitHub release

### CODE REVIEW

**Review by samiwel — COMMENTED** [review]
The changes look good to me, but I'm holding off on approving the PR until we've tested the deploy on release.

**Comment by jonnyshaw89** [comment]
@samiwel Yes, good to wait until we can test it all properly

File: deploy.yml [hunk 1]
```diff
@@ -158,25 +159,26 @@ jobs:
 
           eq_author_tag=$(cat ../tag)
           echo "{\"APPLICATION_VERSION\":\""$eq_author_tag"\"}" > ../../build_args/args
+          cp -R ../eq-author-app/* ../../extracted-eq-author-app
```
Comment (samiwel) [inline deploy.yml]: What's the reason for having to copy everything into the directory above? Isn't it simpler to go up one directory for the `build_args_file`?

**Review by samiwel — CHANGES_REQUESTED** [review]

File: deploy.yml (hunk 1 above)
Comment (jonnyshaw89) [reply deploy.yml]: We are 2 folders deep and the output folders get created at the root

Comment (jonnyshaw89) [reply deploy.yml]: We have to copy it as we extract it from the `tar.gz`

Comment (jonnyshaw89) [reply deploy.yml]: maybe we dont need to `cd eq-author-app` thou

**Review by samiwel — APPROVED** [review]

**Review by SamGodwin2 — APPROVED** [review]

**Pull request merged**